src/common/libutil/llog.h

```diff
@@ -19,6 +19,36 @@
  *
  * Copyright (c) 2010-2013 David Herrmann <dh.herrmann@gmail.com>
  * Dedicated to the Public Domain
+ *
```

Comment:
Rebase error?  (llog.h copyright update belongs in a different commit)

Reply:
oh, dang... :angry: